METRON-1019: Metron 0.4.0 manual installation guide for CentOS 6

lvets opened the pull request:
## Contributor Comments
This is a PR to METRON-1019.
Is adds a manual installation guide for Metron 0.4.0 on CentOS 6 under a new directory 'other-examples'. Additionally, it explains how to install MySQL for the REST UI as well as Apache NiFi for log ingestion.
No code changes are included, this is purely documentation.

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  


In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel). 
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?


### For code changes:
- [ ] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [ ] Have you included steps or a guide to how the change may be verified and tested manually?
- [ ] Have you ensured that the full suite of tests and checks have been executed in the root incubating-metron folder via:
  ```
  mvn -q clean integration-test install && build_utils/verify_licenses.sh 
  ```

- [ ] Have you written or updated unit tests and or integration tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

ottobackwards commented on metron-deployment/other-examples/README.md:
```diff
@@ -0,0 +1,3 @@
+# Other Example Deployments
```
What I would want to see in this readme is some kind of disclaimer, but I'm not sure what it would be.

"these examples have been vetted and tested by the metron community as of the time of entry.  Things may change however, and if there are problems with these steps, please log a jira"

Also, should there be a date on each example, to show when it was entered?
Should there be a revision log?

lvets commented:
Changed the wording in the README.md file to manage expectations.
Added month + year.
There's a version number in the raw source of "Manual_Install_CentOS6.md".

ottobackwards commented:
+1.  I believe this is positioned with the correct expectations, and content wise has been worked out on the list ( where this contribution was indeed solicited ).

Would like to hear from @JonZeolla before committing.
